server/app/auth/oidc/applicant/IdcsProvider.java

```diff
@@ -0,0 +1,60 @@
+package auth.oidc.applicant;
```

Comment:
Not blocking: Since git doesn't really do a very good job of preserving the file this was renamed from, it might make for an easier review if the renames were done mechanically in one PR and then a follow-on PR updated the logic. That way, we'd only be reviewing what changed.

Reply:
Move the file changes to https://github.com/seattle-uat/civiform/pull/2664 (which needs to get merged before this)